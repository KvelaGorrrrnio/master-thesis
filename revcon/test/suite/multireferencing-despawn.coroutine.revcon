@coroutine

coroutine main() {
  local int x 0
  local int y 0
  spawn t test (x,y)
  despawn t test (x,x)
  delocal int x 0
  delocal int y 0
}

coroutine test() with (int a, int b) {
  skip
}
