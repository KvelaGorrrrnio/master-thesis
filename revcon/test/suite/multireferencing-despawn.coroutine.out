The variable x was referenced multiple times as args at (7,3).
